### PR TITLE
Implement `Packable` for tuples

### PR DESCRIPTION
**Pull request by pvdrz**

### CODE REVIEW

File: packable/packable/CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.X.X - 2022-XX-XX
 
+### Added
+
+- Implement `Packable` for tuples;
```
Comment (thibault-martinez) [inline packable/packable/CHANGELOG.md:27]: Anything else planned soon or should we do a version for it ?

Comment (thibault-martinez) [inline packable/packable/CHANGELOG.md:27]: Anything else planned soon or should we do a version for it ?

Comment (pvdrz) [reply packable/packable/CHANGELOG.md:27]: yeah I've planned to do the `(un)?pack_with` thingy

Comment (thibault-martinez) [reply packable/packable/CHANGELOG.md:27]: What's that again ?

**Review by grtlr — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**